app/src/components/idpService.js

```diff
@@ -159,9 +159,10 @@ class IdpService {
             const filterName = f.name;
             const paramName = f.param;
             const value = params[paramName];
-            if ('exact' in f) {
+            if ('exact' in f || 'caseSensitive' in f) {
               const exact = f.exact;
-              q.modify(filterName, value, exact);
+              const caseSensitive = f.caseSensitive;
+              q.modify(filterName, value, exact, caseSensitive);
```

Comment:
I have to admin I'm struggling to understand this whole method, but it seems that you're checking for 'exact' **or** 'caseSensitive', and then setting values for both of them. What if only one is defined? Should there be two if clauses here, one for each value?

Reply:
And having said that, Code Climate is giving complaints about the complexity and length of the idpService methods - this change will increase that. I'm hoping we an get better reporting from Code Climate so that this is flagged in the PRs, but just an FYI.

Reply:
I had the same thought when I looked at it. We should have defaults `exact = false, caseSensitive = true` and only override them if they've been set. I'm not sure if you'll get the expected query result passing in `undefined` for `caseSensitive`.